manifests/config.pp

```diff
@@ -22,15 +21,72 @@
     mode    => '0600',
   }
 
-  file {'/etc/httpd/conf.d/pulp.conf':
+  apache::vhost { 'pulp-https':
+    priority                   => '05',
+    docroot                    => '/srv/pulp',
+    port                       => 443,
+    servername                 => [$::fqdn],
+    serveraliases              => [$::hostname],
+    add_default_charset        => 'UTF-8',
```

Comment:
If we keep it, we'd need to bump the metadata requirement, however, due to how we use this module currently I'd prefer to stay 1.1.0 compatible.